wiki/markdown/curl-examples.md

```diff
@@ -321,7 +321,16 @@ Example curl commands to these endpoints [can be found here](https://github.com/
     <tr>
         <td>/subscriptions</td>
         <td>GET</td>
-        <td>Fetches all available subscriptions from the back-end</td>
+        <td>
+            Fetches all available subscriptions from the back-end.
+            Selected subscriptions can be fetched by using the optional subscriptionNames parameter
+            and a comma separated list e.g. subscriptionNames={name1},{name2}
```

Comment:
Perhaps leave out the curly braces in the example to be more clear? How about "?subscriptionNames=MyFirstSubscription,MySecondSubscription"